packages/git-raw-commits/index.js

```diff
@@ -52,38 +52,31 @@ function gitRawCommits (rawGitOpts, rawExecOpts) {
     gitOpts.debug('Your git-log command is:\ngit ' + args.join(' '))
   }
 
-  var isError = false
-
   var child = execFile('git', args, {
     cwd: execOpts.cwd,
     maxBuffer: Infinity
+  }, function (err) {
+    if (err != null) {
+      readable.emit('error', err)
+    }
+
+    readable.emit('close')
   })
 
   child.stdout
     .pipe(split(DELIMITER + '\n'))
     .pipe(through(function (chunk, enc, cb) {
       readable.push(chunk)
-      isError = false
 
       cb()
     }, function (cb) {
       setImmediate(function () {
-        if (!isError) {
-          readable.push(null)
-          readable.emit('close')
-        }
+        readable.push(null)
 
         cb()
       })
     }))
 
-  child.stderr
-    .pipe(through.obj(function (chunk) {
-      isError = true
-      readable.emit('error', new Error(chunk))
-      readable.emit('close')
-    }))
```

Comment:
The error value passed to `execFile` callback is checked instead of checking STDERR.

Is there a reason STDERR was checked instead?  If yes, this needs to stay and maybe a test should be added for whatever that use case is.